app/src/main/java/com/bc/gordiansigner/ui/account/AccountRecyclerViewAdapter.kt

```diff
@@ -34,6 +34,14 @@ class AccountRecyclerViewAdapter(
         notifyItemInserted(items.size - 1)
     }
 
+    fun update(item: KeyInfo) {
+        items.firstOrNull { it == item }?.let {
```

Comment:
I prefer using the `indexOf()` instead and check value return is not `-1` for doing the logic inside. By that way, we need only one time for searching.

Reply:
I think this way there is one-time searching also, the iterator will be broken when it found the first item so complexity is only O(ln(n))?

Reply:
@hvthhien If we found an item in first iterator `items.firstOrNull` so the `let` function is invoked and then you have to do the second iterator to find the index of item `notifyItemChanged(items.indexOf(it))`

Reply:
ahh, okay, I see, that's right 👍